doc/src/modules/solvers/ode.rst

```diff
@@ -81,15 +81,17 @@ factorable
 
 1st_homogeneous_coeff_best
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::ode_1st_homogeneous_coeff_best
+.. autoclass:: sympy.solvers.ode.single::HomogeneousCoeffBest
 
 1st_homogeneous_coeff_subs_dep_div_indep
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::ode_1st_homogeneous_coeff_subs_dep_div_indep
+.. autoclass:: sympy.solvers.ode.single::HomogeneousCoeffSubsDepDivIndep
+   :members:
 
 1st_homogeneous_coeff_subs_indep_div_dep
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::ode_1st_homogeneous_coeff_subs_indep_div_dep
+.. autoclass:: sympy.solvers.ode.single::HomogeneousCoeffSubsIndepDivDep
+   :members:
```

Comment:
Why do some autoclass blocks have members but others not?

Is it useful to show the members?

Reply:
official documentation says if we keep the :members: block then it will include docstrings of member functions also. but in previous class like Factorable, initially it was not there but someone made commit stating 'expanding documentation', so I thought it is the practice to be followed.